File: prh-service/src/main/java/com/accountor/prh/domain/CompanyDetailsDto.java
package com.accountor.prh.domain;

import java.time.LocalDate;

public record CompanyDetailsDto(
        String businessId,
        String name,
        LocalDate registrationDate,
        String websiteUrl,
        String street,
        String city,
        String postalCode,
        String mainBusinessLineCode,
        String mainBusinessLineDescription) {
}

File: web/src/types/CompanyDetailsDto.ts
export interface CompanyDetailsDto {
    businessId: string;
    name: string;
    registrationDate: string;
    websiteUrl: string | null;
    street: string | null;
    city: string | null;
    postalCode: string | null;
    mainBusinessLineCode: string | null;
    mainBusinessLineDescription: string | null;
}